satellite/payments/stripepayments/stripe.go

```diff
@@ -8,11 +8,11 @@ import (
 	"fmt"
 	"time"
 
-	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go"
 	"github.com/stripe/stripe-go/client"
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
-	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+	"gopkg.in/spacemonkeygo/monkit.v2"
```

Comment:
Unnecessary change

Reply:
Yeah, it looks like it's not using goimports. Usually you name the import explicitly if it doesn't match the path.